dev/com.ibm.ws.messaging.comms.server/src/com/ibm/ws/jfap/inbound/channel/CommsServerServiceFacade.java

```diff
@@ -460,4 +529,12 @@ int getConfigured_wasJmsSSLPort() {
     String getConfigured_Host() {
         return host;
     }
+    
+    /**
+     * Query if Netty has been enabled for this endpoint
+     * @return true if Netty should be used for this endpoint
+     */
+    public boolean useNetty() {
+    	return _useNettyTransport;
```

Comment:
I am not a fan of the "Transport" postfix, there are lots of other places where just 'Netty' is used and the transport layer...is that really Netty or what is underneath like NIO, AIO, Epoll etc